### ISSUE TITLE
Layout locking

### ISSUE THREAD

**Issue by myangelkamikaze** [gre4bee/ElectronicObserver#79]
Layout locking doesn't work.

**Comment by myangelkamikaze** [comment]
Layout locking is currently implemented by setting the grid splitter size to 0. It's still possible to close views, float them or resize EO itself. We'll be listening to feedback to see if this is good enough or needs extra work.

**Comment by y-jin-yoshihiko** [comment]
I want the lock to disable this function.

・Window close button
・Docking & Separation function
・Auto-hide button

**Comment by NaokiSato102** [comment]
> I want the lock to disable this function.
> 
> ・Window close button ・Docking & Separation function ・Auto-hide button

I think so

**Comment by NaokiSato102** [comment]
![image](https://user-images.githubusercontent.com/27545766/148639860-601948e7-5840-4770-961c-7e05c56323eb.png)
It's updated, but docking still works.

**Comment by myangelkamikaze** [comment]
Yeah, disabling docking is a bit harder, we're still looking at how to do that one.

**Comment by NaokiSato102** [comment]
Yeah, I'll be waiting.
